src/views/atoms/richTextEditor/richTextEditor.js

```diff
@@ -0,0 +1,64 @@
+import React, {Component} from 'react';
+import PropTypes from "prop-types";
+
+import ClassicEditor from '@ckeditor/ckeditor5-build-classic';
+import CKEditor from '@ckeditor/ckeditor5-react';
+import EditorPreview from "../editor-preview/editor-preview";
+
+import './richTextEditor.css';
+
+class RichTextEditor extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      data: '',
+    };
+  }
+
+  componentWillMount() {
+    this.setState({data: this.props.data})
+  }
+
+  render() {
+    const {isEditing,i18n} = this.props;
+
+    return (
+      <div className={'rich-text-editor'}>
+
+        {(!isEditing &&
+          <EditorPreview
+            className={'editor-preview'}
+            data={this.state.data}
+            onClick={this.props.onToggleEditing}/>)}
```

Comment:
I would destruct every prop and state variable, such as `onToggleEditing`, on top of render method so the DOM elements would be cleaner :)